bans-core/src/main/java/space/arim/libertybans/core/config/Formatter.java

```diff
@@ -355,13 +373,45 @@ String formatRelative(long diff) {
 		}
 		return joiner.toString();
 	}
+
+	String formatRelativeSimple(long diff) {
```

Comment:
Ideally there should be a unit test for this. The current tests will ensure that it does not throw an exception, but for completeness' sake, we should verify the output as well.

While a small unit test would be ideal I won't consider it a barrier to merging. It doesn't have to be a large test; also, FormatterTest should have most of the setup done for you.